### PR TITLE
Remove code coverage tab

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/airtap/airtap/issues/202

### CODE REVIEW

File: client/reporter.js [hunk 1]
```diff
@@ -194,9 +146,6 @@ Reporter.prototype.done = function (err) {
 
   // add coverage tab content
   if (window.__coverage__) {
-    var coverageTab = document.getElementById('code-coverage-tab')
-    coverageTab.innerHTML = '<iframe frameborder="0" src="/airtap/coverage"></iframe>'
```
Comment (ralphtheninja) [inline client/reporter.js]: @vweevers A bit unsure of this but I'm guessing it's not needed anymore. Also, what actually defines the `__coverage__` global?

**Comment by ralphtheninja** [comment]
Looks like this now:

screenshot from 2018-07-06 15-38-50

I figured the tabs aren't needed anymore.

File: client/reporter.js (hunk 1 above)
Comment (vweevers) [reply client/reporter.js]: Yeah, we don't need that. As for the `__coverage__` global, that's populated by the `browserify-istanbul` transform.

File: client/index.css [hunk 2]
```diff
@@ -32,26 +32,6 @@ html, body {
...
-    color: black;
-    height: 30px;
-    line-height: 30px;
-}
-
-#tab-selector a.selected {
-    background-color: #e1e1e1;
-}
-
 #zuul, .tabs, .tab {
```
Comment (vweevers) [inline client/index.css]: Can these selectors (`.tabs, .tab`) be removed as well?

Comment (vweevers) [reply client/index.css]: Yes, they can. @ralphtheninja I'll take over this branch if you don't mind

**Pull request merged**